OasisDEX/oasis-borrow
# Gf/stop loss confirmation screen

# [Stop Loss Order Confirmation Screen](https://app.shortcut.com/oazo-apps/story/5353/stop-loss-add-order-summary-confirmation-screen)
  
## Changes 👷‍♀️
- Added a screen in the auto buy / auto sell flow for confirmation before confirming an order. 
  
## How to test 🧪
- Go through the add auto buy set up process
- You will see an additional screen in the flow
